src/content/developers/docs/networks/index.md

```diff
@@ -49,7 +56,13 @@ Goerli is a proof-of-stake testnet. It is expected to be maintained long-term as
 
 #### Sepolia {#sepolia}
 
-Sepolia is a proof-of-stake testnet. Although Sepolia is still running, it is not currently planned to be maintained long-term. Before undergoing The Merge in June 2022, Sepolia was a proof-of-work testnet.
+Sepolia is a proof-of-stake testnet. Sepolia will be maintained long-term, and is recommended for users and developers who want a lighter-weight chain to sync to and interact with. Before undergoing The Merge in June 2022, Sepolia was a proof-of-work testnet.
```

Comment:
I think making claims about the state weight is not too useful. Would be better to say in so many words that sepolia is more similar to goerli pre-merge because it has a fixed set of professional validators which tends to lend itself to a high uptime for the chain. We also have mechanisms to mint more sepolia so it will always be easier to retrieve sepEth than goerli eth. I think this should be the preferred testnet for developers.